x-pack/plugins/aiops/server/routes/error_wrapper.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { boomify, isBoom } from '@hapi/boom';
+import { ResponseError, CustomHttpResponseOptions } from '@kbn/core/server';
+
+export function wrapError(error: any): CustomHttpResponseOptions<ResponseError> {
```

Comment:
A duplicate of the function in ML.
Could possibly be moved to a package if we ever need it in another plugin.

Reply:
The same is already in multiple plugins. We already have `@kbn/ml-error-utils` and could move it there, but I think for now this package is only used for client side utils, so we need to refactor that to be split into server/client side code to not unnecessarily blow up client side bundle sizes.